packages/atlas/src/views/viewer/VideoView/VideoView.tsx

```diff
@@ -164,6 +173,14 @@ export const VideoView: React.FC = () => {
     setDetailsExpanded((prevState) => !prevState)
   }
 
+  const aspectRatio = useMemo(() => {
+    if (!video || !video.mediaMetadata) {
+      return DEFAULT_ASPECT_RATIO
+    }
+    const { pixelHeight, pixelWidth } = video.mediaMetadata
+    return (pixelHeight || 0) / (pixelWidth || 0)
```

Comment:
0 may not be the best default when it comes to division 😅 Maybe just default to `DEFAULT_ASPECT_RATIO` if any of the values is missing?